srcpkgs/signond/template

```diff
@@ -4,7 +4,7 @@ version=8.60
 revision=2
 wrksrc="signond-VERSION_${version}-4d195e4dc7a47ff5cb51e36a83d4d05808c5befe"
 build_style=qmake
-configure_args="LIBDIR=/usr/lib"
+configure_args="LIBDIR=/usr/lib CONFIG+=rtti"
```

Comment:
Does the current version crash? If yes, please bump revision.

Reply:
yes, the actual version crashes. (segfault error)